appveyor.yml

```diff
@@ -52,6 +52,7 @@ matrix:
     - OTP_VERSION: 21.2
 
 init:
+  - ps: Update-AppveyorBuild -Version "$env:appveyor_build_version-$($env:appveyor_repo_commit.substring(0,7))"
```

Comment:
From https://www.appveyor.com/docs/environment-variables/
> `APPVEYOR_BUILD_VERSION` - build version

(Uppercase.) 

I [see](https://www.appveyor.com/docs/build-configuration/#build-versioning) build version is `version:`.

Reply:
Does lowercase/uppercase matter?